src/components/CommentBox.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+
+export default function CommentBox({
+  handleSubmitComment,
+  id,
+  comments,
+  commentValue,
+  setCommentValue,
+}) {
+  return (
+    <CommentBoxContainer>
+      <SubHeading>Leave a comment:</SubHeading>
+      <CommentList role="list">
+        {comments &&
+          comments.map(comment => (
+            <CommentItem key={comment.commentId}>
+              {comment.commentText}
+            </CommentItem>
+          ))}
+      </CommentList>
+      <label htmlFor="commentBox"></label>
+      <textarea
+        id="commentBox"
+        name="commentBox"
+        type="text"
+        placeholder="Add a comment..."
+        rows="10"
+        cols="5"
+        maxLength="500"
+        value={commentValue}
+        onChange={e => setCommentValue(e.target.value)}
+      />
+      <CommentButton
+        type="submit"
+        disabled={commentValue ? false : true}
+        onClick={() => handleSubmitComment(commentValue, id)}
+      >
+        Comment
+      </CommentButton>
+    </CommentBoxContainer>
+  );
+}
+
+const CommentBoxContainer = styled.div`
+  margin-top: 20px;
+  display: grid;
+  gap: 15px;
+  min-width: 220px;
+  max-width: 70%;
+
+  textarea {
+    width: 100%;
+    border: 1px solid var(--bg-color-secondary);
+    border-radius: 5px;
+    padding: 3px 5px;
+
+    &:focus {
+      outline: none;
+      border-color: var(bg-color-button);
+    }
+  }
+`;
+
+const SubHeading = styled.p`
+  font-weight: bold;
+  color: var(--font-color-headlines-bright);
+`;
+
+const CommentList = styled.ul`
+  list-style: none;
+`;
+
+const CommentItem = styled.li`
+  color: var(--font-color-text-bright);
+  margin-top: 5px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  border-bottom: 1px solid var(--font-color-text-bright);
+`;
+
+const CommentButton = styled.button`
```

Comment:
If you use this button more than once and also in other components you can think about making an own button component (maybe only a styled component). This would be good to reduce repetitive code.

Reply:
Yes I am using different buttons in many components. A button component with different variants will be part of the refactoring I want to do :)